site/src/pages/api/uploadUserAvatar.api.ts

```diff
@@ -110,7 +111,7 @@ export default createAuthenticatedHandler<
     }
   });
 
-export const config = {
+export const config: PageConfig = {
```

Comment:
Unrelated bonus – helps make sure that the shape of `config` is correct.

I first thought I had to change config here but then discovered https://vercel.com/docs/project-configuration#project-configuration/functions